VideoLocker/res/layout/navigation_drawer_container.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/slider_menu"
+    android:layout_width="304dp"
+    android:layout_height="match_parent"
+    android:layout_gravity="left|start"
```

Comment:
This is the default gravity (for views and layouts), so this definition is unnecessary.